### PR TITLE
Added char-n-gram + tdIdf approach

### PR DESCRIPTION
**Pull request by LiubovVdovina**

### CODE REVIEW

File: projects/char_n_gram.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import pandas as pd
+
+from sklearn import preprocessing, svm
+from sklearn.feature_extraction.text import TfidfVectorizer
+from sklearn.metrics import classification_report
+
```
Comment (AlexDel) [inline projects/char_n_gram.py:7]: Please add a random seed for results to be always deterministic, https://www.mikulskibartosz.name/how-to-set-the-global-random_state-in-scikit-learn/

**Review by AlexDel — CHANGES_REQUESTED** [review]
Please add random seed.

**Review by AlexDel — APPROVED** [review]
